### DISCUSSION TITLE
Avoid remapping of category ids

### DISCUSSION THREAD

**Discussion by deepaksinghcv** [facebookresearch/detectron2#2672] [Q&A] [answered]
Hello community,

I'm trying to build an incremental object detector. I'm performing the experiment on PASCAL VOC classes (I have converted PASCAL VOC to COCO).
During the increment step, the dataset for the new class will be available, but the category id will be 15(say). 
By default, the COCO dataloader performs the remapping of category ids (In this case, 15 -->0).
Does anyone has any solution or feedback on how to avoid this remapping, since the downstream coco evaluation shouldn't get affected.

Thanks in advance.

**Comment by ppwwyyxx** [comment, marked as answer]
You can declare unused categories in the COCO-format json file.

**Reply by deepaksinghcv** [reply]
Just to confirm.
You mean to have the images and annotations of category 15 only, but in `categories` include the previous category ids along with category id 15.
